pyproject.toml

```
@@ -25,15 +25,17 @@ classifiers = [
     "Intended Audience :: Telecommunications Industry",
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.9",
 ]
 requires = [
     "structlog~=22.1.0",
     "colorama~=0.4.3",
     "redis~=4.4.2"
 ]
 description-file = "README.md"
-requires-python = ">3.9"
+requires-python = ">3.8"
```

I would prefer showing the required python version then above a previous version
```suggestion
requires-python = ">=3.9"
```